tests/expected_traces.py

```diff
@@ -114,23 +114,23 @@
 """
 PARITY_CREATE_TRACE_EXPECTED_OUTPUT = """
 CALL: 0x76E2cFc1F5Fa8F6a5b3fC4c8F4788F0116861F9B.<0x61b69abd> [417274 gas]
-├── CREATE: 0xFEB4acf3df3cDEA7399794D0869ef76A6EfAff52 [None gas]
```

Comment:
There is supposed to be gas here actually but handling the case when it is `None` is still nice.
But currently debugging and will get the gas working for parity as well as the "failed"
TBD